examples/npm_deps/BUILD.bazel

```diff
@@ -210,3 +210,24 @@ uvu_test(
         ":uvu_spec",
     ],
 )
+
+#######################################
+# Use case 6: depend on a hoisted npm package that wasn't a direct dependency package.json
```

Comment:
would be nice to say which direct dependency has the `ms` dependency

Reply:
✅